PurchasesCoreSwiftTests/Mocks/MockSKProduct.swift

```diff
@@ -32,27 +32,26 @@ class MockSKProduct: SKProduct {
         return mockPrice ?? 2.99 as NSDecimalNumber
     }
 
-    @available(iOS 11.2, *)
+    @available(iOS 11.2, macCatalyst 13.0, macOS 10.13.2, tvOS 11.2, *)
```

Comment:
added all of these since we'd like tests to be runnable from other OS versions. 
watchOS is a notable omission here, but it's only because we support 6.2+, which already includes this, so it should be caught by the `*` I believe